### ISSUE TITLE
[EMU] Autologin fails at character select

### ISSUE THREAD

**Issue by SirCabby** [macroquest/macroquest#617]
When trying to connect to the EMU server "Project Lazarus", I was unable to get it past the character select.  This is what I think is happening.

The docs for MQ2AutoLogin here: https://docs.macroquest.org/plugins/core-plugins/mq2autologin/?h=mq2autologin
mention that Project Lazarus should be referenced as such:
`Project Lazarus=[] Project Lazarus`

However when doing this, it was unable to find the server until I changed the key to remove the space, and also to remove the "[] ".  I was able to get it to go beyond server select by using:
```
[Servers]
laz=Project Lazarus

[redacted account name]
Password=redacted pw
Server=laz
Character=redacted
```

But as soon as I reach the character select screen, it immediately punts me back to server select.

I modified the plugin code to remove the check for correct server name at the character select screen, which made this plugin work again.  This is the removed code I had to do was in StateMachine.cpp:

```
//if (GetServerShortName()[0] == 0 || !m_record
			//	|| (!m_record->serverName.empty() && !ci_equals(GetServerShortName(), m_record->serverName)))
			//{
			//	// wrong server, need to quit character select to get to the server select window
			//	if (pCharacterListWnd)
			//		pCharacterListWnd->Quit();

			//	transit<Wait>();
			//}
			//else
			//{
```

Ideally we'd probably want this code working again, or not?  Just not working by default out of the gate.

**Comment by MikeD-CCF** [comment]
There are two things that you have that are different than the Directions: 
1: you are not using [Servers: Host=login.eqemulator.net] 
2: when Host File is login.projecteq.net You need to use: Project Lazarus=Project Lazarus

The New Auto Login Requires,  Server ShortName = Server LoginScreen Exact Name 
A: When At the Server Select Screen it uses "Server LoginScreen Exact Name" 
B: At Character Select it checks Current "Server Shortname" If its not exact it kicks you out so to log into the correct server 

So In Short if you are not getting past Server LoginScreen then you have Server LoginScreen Wrong 
If you are not getting past Character Select Screen and are auto Kicked back to Server then Server ShortName is wrong

**Comment by SirCabby** [comment]
@MikeD-CCF Thanks for the reply.

I had tried using the `: Host=login.eqemulator.net` before and didn't see a difference and ended up removing it.  My hosts file is using:
```
[LoginServer]
Host=login.projecteq.net:5999
```

I tried what you suggested, let me know if I missed something:
```
[Settings]
UseMQ2Login=0
UseStationNamesInsteadOfSessions=1

[Profiles]
DefaultEQPath=C:\Project Lazarus

[Servers: Host=login.eqemulator.net]
Project Lazarus=Project Lazarus

[snip]
Password=snip
Server=Project Lazarus
Character=snip
```

When doing this, it hangs at the server select screen as I originally mentioned.  I also tried `[Servers: Host=login.projecteq.net:5999]` with same result.

**Comment by SirCabby** [comment]
Got it working with help from Knightly in discord.  Apparently I needed to leave it just [Servers] for PEQ login.  Can this issue be used to update documentation to capture this combination and explanation?  Thanks

**Comment by SirCabby** [comment]
Knightly updated the docs, closing and thanks
